### PR TITLE
Submitting evaluation of Openfoodfacts

### PR DESCRIPTION
**Pull request by Flydx1234**

### CODE REVIEW

File: openfoodfacts_evaluation_2.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+
+**Evaluating Person or Team**: David, Flydx1234
+
+
+## Project Data
+
+1. Project description: <br>
+
+Open Food Facts is a food products database. It basically contains information on foods from all around the world
```
Comment (joannakl) [inline openfoodfacts_evaluation_2.md]: please, indent this so that it is part of item one - this should also fix numbering in the following two items

Comment (Moonsolol) [reply openfoodfacts_evaluation_2.md]: Okay, I think I fixed the issue.

Comment (Moonsolol) [reply openfoodfacts_evaluation_2.md]: wait sorry wrong pull request lol

Comment (Flydx1234) [reply openfoodfacts_evaluation_2.md]: Fixed!

**Pull request merged**